src/main/java/com/woowacourse/lotto/dao/UserLottoDAO.java

```diff
@@ -0,0 +1,63 @@
+package com.woowacourse.lotto.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.woowacourse.lotto.domain.Lotto;
+import com.woowacourse.lotto.domain.LottoNumber;
+
+import static com.woowacourse.lotto.domain.LottoNumber.NUMBER_OF_LOTTO;
+
+
+public class UserLottoDAO {
+	private final Connection connection;
+
+	public UserLottoDAO(Connection connection) {
+		this.connection = connection;
+	}
+
+	public void addUserLotto(Lotto lotto, int round) throws SQLException {
+		String query = "insert into user_lotto(first_number, second_number, third_number, fourth_number, fifth_number, sixth_number, winning_lotto_fk)" +
+				"VALUES(?,?,?,?,?,?,?)";
+		List<Integer> lottoNumber = new ArrayList<>();
+		lotto.getNumbers().stream().forEach(number -> lottoNumber.add(number.getName()));
+
+		PreparedStatement pstmt = connection.prepareStatement(query);
+		for (int i = 1; i <= lottoNumber.size(); ++i) {
+			pstmt.setInt(i, lottoNumber.get(i - 1));
+		}
+		pstmt.setInt(7, round);
+		pstmt.executeUpdate();
+	}
+
+	public Lotto findUserLottoById(int userLottoId) throws SQLException {
+		String query = "select first_number, second_number, third_number, fourth_number," +
```

Comment:
나중에 필드 갯수가 늘어날 때를 대비해서 읽기 쉽고 수정하기 쉽게 작성해두는게 좋아요
```suggestion
		String query = String.join("\n",
				"select first_number",
				"     , second_number",
				"     , third_number",
				"     , fourth_number",
				"     , fifth_number",
				"     , sixth_number",
				"  from user_lotto",
				" where id = ?");
```

Reply:
네! 문장을 나눠서 쿼리문을 나타내니 가독성이 높아졌습니다! 감사합니다~